public/app/features/alerting/state/alertDef.ts

```diff
@@ -81,7 +81,10 @@ function createReducerPart(model: any) {
 }
 
 function getStateDisplayModel(state: string) {
-  switch (state) {
+  const normalizedState = state.toLowerCase().replace(/_/g, '');
+
+  switch (normalizedState) {
```

Comment:
@kylebrandt I see there is an additional state (Error) which is not handled here. Which one should represent it?

Reply:
I feel like it should be visually different from "alerting" but can't think of how (someone on UX might have an idea?). 

If it isn't going to be different for now, then the same as state-class and icon class as alerting I would imagine (except with different text).

Reply:
@kylebrandt ok, it's now added. If this looks good in your opinion, we should be good to get it on main.